run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+docker run \
```

Comment:
Docker knows of `--env`, which can be passed the location of an env file so we wouldn't need to repeat the keys in the command.